src/server/routes/parcel.routes.ts

```diff
@@ -0,0 +1,66 @@
+import type { FastifyInstance, RouteOptions } from 'fastify';
+import { Parcel } from '@relaycorp/relaynet-core';
+
+import type { PluginDone } from '../../utilities/fastify/PluginDone.js';
+import { bufferToArrayBuffer } from '../../utilities/buffer.js';
+import { HTTP_STATUS_CODES } from '../../utilities/http.js';
+
+export default function registerRoutes(
+  fastify: FastifyInstance,
+  _opts: RouteOptions,
+  done: PluginDone,
+): void {
+  fastify.removeAllContentTypeParsers();
+  fastify.addContentTypeParser(
+    'application/vnd.awala.parcel',
+    { parseAs: 'buffer' },
+    (_request, payload, next) => {
+      next(null, payload);
+    },
+  );
+
+  interface Request {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    readonly Body: Buffer;
```

Comment:
this is needed for the typescript to read the body as a buffer

Reply:
Shall I try and find a better way of doing this or go to the next task?

Reply:
The only problem I see with this is that you're redeclaring an interface every time that function is called. I'd just move it outside the function and call it something like `BinaryRequest`.

Reply:
In my comment, I was referring to the disabling the lint. If that's ok I'll just implement it exactly the same way as in the example

Reply:
OK, that was ambiguous then, but I understand it now. Yes, please do what I did in Pong.